### PR TITLE
fix data structure to merge

### PR DESCRIPTION
**Pull request by roothybrid7**
I had a problem with the way the data was merged and I fixed it.

### CODE REVIEW

File: packages/next-rest-framework/src/shared/open-api.ts [hunk 1]
```diff
@@ -97,6 +97,13 @@ export interface NrfOasData {
   schemas?: Record<string, OpenAPIV3_1.SchemaObject>;
 }
 
+export const isNrfOasData = (x: unknown): x is NrfOasData => {
+  if (typeof x !== 'object' || x === null) {
+    return false;
+  }
+  return 'paths' in x;
+};
```
Comment (blomqma) [inline packages/next-rest-framework/src/shared/open-api.ts:105]: Could use move this utility to the same file as the CLI script as this is only used by the CLI commands, thanks.

**Review by blomqma — CHANGES_REQUESTED** [review]
Thanks for discovering and fixing this! I had one suggestion for moving the utility function to the CLI script file. Could you also remove the first commit of updating the lock file as I'm about to do some version upgrades for various packages soon which will update the lock file. The updated lock file version also fails on the CI and it should pass if you remove the first commit.

File: packages/next-rest-framework/src/shared/open-api.ts (hunk 1 above)
Comment (roothybrid7) [reply packages/next-rest-framework/src/shared/open-api.ts]: ok. I moved it to the CLI script file.

**Review by blomqma — APPROVED** [review]
💎

**Pull request merged**